### PR TITLE
fix: added base UI font family to the Menu of edgeless

### PR DESCRIPTION
**Pull request by RajeevDash007**
fixed: #3781 
Screenshot (10)
@xell check this out

### CODE REVIEW

**Comment by RajeevDash007** [comment]
@xell @doodlewind 
added font to the Toolbar too.
Screenshot (24)

**Review by doodlewind — APPROVED** [review]
Thanks for the quick response! It's land-able since CI passes and the patch looks pretty neat, but I'd suggest just keep this open for a while, since we are trying the workflow that for styling issues labeled with `mod:style`, we await for the design team to verify the effect. So /cc @xell @hwangdev97 to confirm.

**Review by xell — APPROVED** [review]

File: packages/blocks/src/page-block/edgeless/components/component-toolbar/more-button.ts [hunk 1]
```diff
@@ -58,6 +58,7 @@ export class EdgelessMoreButton extends WithDisposable(LitElement) {
       display: block;
       color: var(--affine-text-primary-color);
       fill: currentColor;
+      font-family: var(--sl-font-sans);
```
Comment (doodlewind) [inline packages/blocks/src/page-block/edgeless/components/component-toolbar/more-button.ts:61]: @RajeevDash007 though this works, but seems that the `--sl-font-sans` is the [Shoelace](https://shoelace.style/) variable that only works in the BlockSuite playground environment (we use Shoelace for debug menu here, which is not included in the exported editor), but we intend to fix this in AFFiNE. So here the correct fix should be using the `--affine` prefixed variable.

**Comment by RajeevDash007** [comment]
@doodlewind check  #3954

**Pull request merged**